src/components/ProfileDropdown.jsx

```diff
@@ -66,66 +65,79 @@ const useStyles = makeStyles((theme) => ({
 export default function ProfileDropdown(props) {
   const classes = useStyles();
   const { signOut, name, email, photo } = props;
+
   const [dropMenu, setDropMenu] = React.useState(null);
+  const handleClick = (event) => setDropMenu(event.currentTarget);
+  const handleClose = () => setDropMenu(null);
 
-  const handleClick = (event) => {
-    setDropMenu(event.currentTarget);
-  };
+  // needs to be updated once there is a working manage profile page
+  const manageProfileLink = '/home';
+
+  function createMenuItem(text, icon, task) {
+    return (
+      <>
+        <MenuItem onClick={task} className={classes.menuListItem}>
+          <ListItemIcon>{icon}</ListItemIcon>
+          <ListItemText primary={text} />
+        </MenuItem>
+      </>
+    );
+  }
 
-  const handleClose = () => {
-    setDropMenu(null);
+  const display = {
+    0: {
+      manageProfile: (
+        <>
+          {EXPERIMENTAL_VERSION ? null : (
+            <a href={manageProfileLink}>Manage Your Profile</a>
+          )}
+        </>
+      ),
+      navigation: (
+        <>
+          {EXPERIMENTAL_VERSION ? null : (
+            <>
+              {createMenuItem('New', <NewIcon fontSize="medium" />)}
+              {createMenuItem('Saved', <BookmarkIcon fontSize="medium" />)}
+              {createMenuItem('Applied', <DoneIcon fontSize="medium" />)}
+            </>
+          )}
+        </>
+      ),
+    },
   };
```

Comment:
As there's no loop I recommend inlining as it was. The things that need to be conditionally rendered can just be inside `{experiment && <>...<>}` statements.

Reply:
Got it done